template/README.md

```diff
@@ -236,6 +236,10 @@ There are several commands that help to run the vagrant-box:
 
 - `make clean` takes down the provisioned box if there is any.
 
+- `make dev` same as `make up` but without the ansible test playbook and custom_ca.
```

Comment:
- This is not quite right. It doesn't skip a playbook, but it skips all tasks within a playbook tagged with `test`. Also add link to https://docs.ansible.com/ansible/latest/user_guide/playbooks_tags.html

- I see it doesn't set `CUSTOM_CA`. I can only imagine that is an error. @zhenik any input on that?

Reply:
@pdmthorsrud is right its missing `CUSTOM_CA`. Its error

Reply:
I suggest you just fix that error in this PR @dangernil